python/ray/serve/replica.py

```diff
@@ -179,6 +184,24 @@ async def handle_request(
             query = Query(request_args, request_kwargs, request_metadata)
             return await self.replica.handle_request(query)
 
+        async def handle_request_java(
```

Comment:
Should we add the doc comment for this method? I really don't know what this method is from the name, maybe it's `handle_java_request`?

Reply:
python serve Function declaration:
```
        @ray.method(num_returns=2)
        async def handle_request(
            self,
            pickled_request_metadata: bytes,
            *request_args,
            **request_kwargs,
        )
```
Java serve call python replica, can't return two output parameters. So we implement `handle_ request_ java` to handle requests from java serve handle.
`handle_java_request`  is clearer. Previously, when the interfaces of Java serve and python serve needed different implementations, we added a suffix `_java` after the function name.  So we continue use `handle_ request_ java`

Reply:
Yeah, but at first we should make code clear and readable. I still think it should be `handle_java_request` or `handle_request_from_java`, instead of handle_request_java.
@simon-mo CC

Reply:
Resolved. Rename to `handle_request_from_java`